Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -291,23 +295,22 @@ i8 Decoder::read_delta_q()
 
 bool Decoder::segmentation_params()
 {
-    auto segmentation_enabled = m_bit_stream->read_bit();
-    if (!segmentation_enabled)
+    m_segmentation_enabled = m_bit_stream->read_bit();
+    if (!m_segmentation_enabled)
         return true;
 
-    auto segmentation_update_map = m_bit_stream->read_bit();
-    if (segmentation_update_map) {
+    m_segmentation_update_map = m_bit_stream->read_bit();
+    if (m_segmentation_update_map) {
         for (auto i = 0; i < 7; i++) {
             m_segmentation_tree_probs[i] = read_prob();
         }
-        auto segmentation_temporal_update = m_bit_stream->read_bit();
+        m_segmentation_temporal_update = m_bit_stream->read_bit();
         for (auto i = 0; i < 3; i++) {
-            m_segmentation_pred_prob[i] = segmentation_temporal_update ? read_prob() : 255;
+            m_segmentation_pred_prob[i] = m_segmentation_temporal_update ? read_prob() : 255;
         }
```

Comment:
You could remove the brackets for those two `for`-loops, but that's not necessarily required of course. :^)